cache/compressed_secondary_cache.h

```diff
@@ -65,14 +77,16 @@ class CompressedSecondaryCache : public SecondaryCache {
       uint32_t compress_format_version = 2);
   virtual ~CompressedSecondaryCache() override;
 
-  const char* Name() const override { return "CompressedSecondaryCache"; }
+  const char* Name() const override { return kCompressedSecondaryCacheName; }
 
   Status Insert(const Slice& key, void* value,
                 const Cache::CacheItemHelper* helper) override;
 
   std::unique_ptr<SecondaryCacheResultHandle> Lookup(
       const Slice& key, const Cache::CreateCallback& create_cb, bool /*wait*/,
-      bool& is_in_sec_cache) override;
+      bool advise_erase, bool& is_in_sec_cache) override;
```

Comment:
Need comment here to explain the requirement for the Lookup() implementation by `advise_erase`.